R/rdeephaven/R/client_wrapper.R

```diff
@@ -53,8 +53,10 @@ Client <- R6Class("Client",
         #' a variable name on the server. See `?TableHandle` for more information.
         #' @param table_object An R Data Frame, a dplyr Tibble, an Arrow Table, or an Arrow RecordBatchReader
         #' containing the data to import to the server.
+        #' @param num_rows Optional integer argument, only used if table_object is an Arrow RecordBatchReader.
+        #' Number of rows in the table represented by the RecordBatchReader.
         #' @return TableHandle reference to the new table.
-        import_table = function(table_object) {
+        import_table = function(table_object, num_rows=NULL) {
```

Comment:
The change in this file makes no sense to me.  If a user wants fewer rows, they should do `head`, `tail`, or `where` to get less rows.  Providing this functionality for just one type of input feels like a bad API decision.  It should work everywhere or nowhere, and I think nowhere makes more sense.